### DISCUSSION TITLE
VitePress vs Vuepress-next

### DISCUSSION THREAD

**Discussion by suneetk92** [vuepress/core#48] [General]
Which one is going to be the future VitePress or Vuepress-next?

**Comment by meteorlxy** [comment]
For now, they are likely to be two different projects. You can check out the documentation of VitePress:

https://vitepress.vuejs.org/#will-this-become-the-next-vuepress-in-the-future

**Comment by Mister-Hope** [comment]
For now, as a developer who has plan to work on vitepress and vuepress2 projects, to me the biggest difference is below:

- vitepress is fast and simple.

  vitepress is built on vite, with native esm support and fast load and hot-load.✔
  
  vitepress DO NOT have plugin system, and it only support local theme now (in `.vitepress/theme`), which means if you want features beyond those vitepress itself  are providing, you have to handle them on your own.❌

  vitepress has supports from Evan You and his teams.✔

- Vuepress is powerful

  Vuepress has it's plugin and theme apis, and has a good ecosystem. You can simply use plugins to add new features to your project.✔

  Currently vuepress2 is on webpack5, which is much more slower compared to vitepress when booting and hmr, ❌but @meteorlxy has plans to support vite.✔

   Evan You has drop vuepress support with his teams, so it's more likely to be a community project by @meteorlxy. ❌But this project has a great code quality, and I think there will be more people like me making helps to this project.

**Reply by meteorlxy** [reply]
VuePress has not been dropped by Vue.js team.

**Reply by meteorlxy** [reply]
Current vuepress-next repo might be merged into vuejs/vuepress

**Reply by Mister-Hope** [reply]
No offense, but in my eyes only you still remain active in vuepress.

In 2020, vuepress solve less than 1/3 of issues and feature request users open.

**Comment by suneetk92** [comment]
So, I believe I am on a right track to start investing time in vuepress-next. :)
Thanks @meteorlxy and @Mister-Hope for your response.

**Comment by tassoman** [comment]
Looks like a vuewars thing 😆 I'm still confused, but my documentation project needs to crank up
